config/docs/options_README.md

```diff
@@ -0,0 +1,264 @@
+e.g.* .2  
+`painting_image_names:` *The names of all input images used for Cell Painting as they are named in CellProfiler.
+You can find image names as part of any measurement column in Image.csv (e.g. columns starting with Width_).  
+e.g.*  
+    - ConA  
+    - Hoechst  
+    - Mito  
+    - SYTO  
+    - WGA  
+`barcoding_prefix:` *The prefix at the start of all input images used for SBS as they are named in CellProfiler.
+You can find image names as part of any measurement column in Image.csv (e.g. columns starting with Width_)  
+e.g.* CorrCycle
+
+## profile:
+*Configuration specific to the steps in 1.generate-profiles.*
+
+`single_cell:`  
+`perform:` *Do you want to perform 0.merge_single_cells step?  
+Set to* true *or* false.  
+
+`force_overwrite:` *Do you want to overwrite any existing data?  
+Set to* true *or* false.  
+
+`prefilter_features:` *Do you want to use the prefilter file created during preprocessing?
+Set to* true *or* false.  
+
+`output_one_single_cell_file_only:` *Do you want to output a single file SAYMORE.
+False is recommended, particularly for large datasets.
+Set to* true *or* false.  
+
+`cell_quality_column:` *The name of the column that contains the cell quality information.
+e.g.* Metadata_Foci_Cell_Quality  
+
+`sanitize_gene_col:` *Do you want to perform sanitization of gene columns? Fixes formatting for gene names if they aren't just GENENAME and/or contain underscores. True is recommended.
+Set to* true *or* false.  
+
+`merge_columns:` *The names of the columns used to combine data.*  
+`image_column:` *The name of the column that contains the image identifier.
+e.g.* ImageNumber
+`linking_compartment:` *The compartment that is used to link all of the measured compartments. e.g.* Cytoplasm  
+`linking_columns:` *The metadata columns for each additional compartment that identify the linking_compartment. If you add additional* `compartments` *above, add* `linking_columns` *here.*  
+`cells:` *e.g.* Metadata_Cytoplasm_Parent_Cells  
+`nuclei:` *e.g.* Metadata_Cytoplasm_Parent_Nuclei  
+`metadata_linking_columns:` *The columns used to merge single cell profiles and metadata.  
+e.g.*  
+    - Metadata_Foci_site  
+    - Metadata_Cells_ObjectNumber
+
+`aggregate:`  
+`perform:` *Do you want to perform 1.aggregate step?  
+Set to* true *or* false.  
+`force_overwrite:` *Do you want to overwrite any existing data?  
+Set to* true *or* false.  
+`operation:` *A string indicating how the data is aggregated. Currently only supports* mean *or the default of* median *. See pycytominer documentation for more information.  
+e.g.* median  
+`features:` *Set to* all *or pass a list of features that should be aggregated. Default of* infer *uses pycytominer utils to infer the features list. Note that pycytominer assumes standard Cell Painting compartments. e.g.* infer  
+`levels`*:*  
+`gene:` *EXPLAIN e.g.*  
```

Comment:
@gwaygenomics

Reply:
In image-based profiling experiments using CRISPR perturbations, we aggregate single cells (average morphology features) in two possible ways (`gene` and `guide`). `gene` aggregation averages all single cells infected with all `guides` identified to target the specific gene.